### DISCUSSION TITLE
Some Astra Urls

### DISCUSSION THREAD

**Discussion by MapGuy11** [iptv-org/community#233] [Show and tell]
@LaneSh4d0w  I have no idea about these
http://64.18.129.2:8002/
http://198.46.161.101:8000/
http://198.255.102.220:8000/
http://149.13.0.69:8000/
http://104.12.46.240:8000/
http://198.255.102.90:8000/
http://198.255.102.196:8000/
http://64.18.131.130:8000/
http://149.13.0.58:9000/
http://198.255.102.198:8000/
https://search.censys.io/hosts/199.38.246.164 - Waay To Many To List
http://69.64.57.138:8000/
http://71.187.29.220:8000/
http://5.254.76.118:8082/
http://141.95.2.3:8000/
http://71.105.16.123:2086/
http://162.212.178.78:8000/

**Comment by BellezaEmporium** [comment]
https://199.38.246.164 = Flussonic server... Sounds like one of the hosts that we have.

**Reply by BellezaEmporium** [reply]
It seems you've found something that belongs to https://vtveee.com/

**Reply by MapGuy11** [reply]
Yeah it said it on the DNS records I didn't even check lol :). I get links for a service not even out yet.

**Reply by MapGuy11** [reply]
@LaneSh4d0w Let me know if you find anything! :)

**Reply by MapGuy11** [reply]
> https://199.38.246.164 = Flussonic server... Sounds like one of the hosts that we have.

There is also some Astra on other ports

**Comment by BellezaEmporium** [comment]
Found out from where the control panels come from, it's from a program called Astra : https://cesbo.com/en

Could be confusing due to the fact that this name is the same as a satellite called Astra.

**Reply by BellezaEmporium** [reply]
Some hosts can be accessed with username/password admin:admin

https://cesbo.com/en/latest/astra

**Reply by MapGuy11** [reply]
Which ones?

**Reply by BellezaEmporium** [reply]
Haven't found one yet, but if someone can get access to it, it's possible.

**Comment by BellezaEmporium** [comment]
http://64.18.129.2:8002/ Hosted on Octon Telecom z.o.o Poland
http://198.46.161.101:8000/ On port 4444, MultiCS, connects with admin:admin (so, badly secured). Seems it's a CCCam service (provides decryption keys for encrypted satellite channels), possesses 18 servers + 1 that is a cache and another one being a Oscam CS378X... Also connects on Astra Control Panel with admin:admin. Sounds hella fishy to me.

One entry : http://198.46.161.101:8000/play/a001/index.m3u8 named "tvi"

http://198.255.102.220:8000/ Not so much information about that one.

http://69.64.57.138:8000/ Seems that someone's playing with lots of stuff. a Jenkins server located on port 18090 and a MySQL DB located on 3306

http://71.187.29.220:8000/playlist.m3u8 Some local channels + Spanish ones

http://71.105.16.123:2086/ Hmm...
![image](https://user-images.githubusercontent.com/30985701/127845430-75fa63b4-7b9c-4132-85d6-34208e556d29.png)

http://162.212.178.78:8000/ IPVTL found on port 8888

**Comment by Dum4G** [comment]
https://github.com/michelotero25/oteroiptv.live/blob/aef7b64a2f747c1677da788889f4fdd322fc269d/Telegram.m3u8

**Reply by BellezaEmporium** [reply]
While those aren't Astra Control URLs, it's a very good source of Paid TV streams it seems...

**Reply by MapGuy11** [reply]
> While those aren't Astra Control URLs, it's a very good source of Paid TV streams it seems...

All 360p though :(
